packages/nextjs/utils/scaffold-eth/networks.ts

```diff
@@ -78,3 +79,16 @@ export function getBlockExplorerTxLink(network: Network, txnHash: string) {
 
   return `${blockExplorerTxURL}/tx/${txnHash}`;
 }
+
+/**
+ * @returns targetNetwork object consisting targetNetwork from scaffold.config and extra network meta data
+ */
+
+export function getTargetNetwork(): chains.Chain & Partial<TChainAttributes> {
```

Comment:
using `Partial<TChainAttributes>` here since there might be the cases where : 
```js
return {
    ...configuredNetwork,
    ...NETWORKS_EXTRA_DATA[configuredNetwork.id] -> undefined,
  };
```

And typescript will suggest the developer to first check if propery mentioned in `TChainAttributes` exists and then use it. 

Suggestion are open 🙌